main.go

```diff
@@ -59,7 +59,11 @@ func main() {
 	configFile := command.String("config", "./.ergo", "Set the services file")
 	command.Parse(os.Args[2:])
 
-	config.Services = proxy.LoadServices(*configFile)
+	services, err := proxy.LoadServices(*configFile)
+	if err != nil {
+		fmt.Printf("Could not load services: %v\n", err)
```

Comment:
I believe that here a log.Printf would be more appropriate as this might be seen as a warning.
Warnings ans errors should go to stderr and log does that. fmt.Printf writes to stdout.